_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: distill
+title: "Lecture 8: Causal Discovery and Inference"
+description: <insert description here>
+date: <insert date here>
```

Comment:
Insert description and date here. See https://raw.githubusercontent.com/sailinglab/pgm-spring-2019/master/_posts/2019-02-13-lecture-09.md for an example.

Reply:
Thanks @alshedivat for the fix